quai/backend.go

```diff
@@ -261,6 +261,10 @@ func New(stack *node.Node, p2p NetworkingAPI, config *quaiconfig.Config, nodeCtx
 
 	// Set the p2p Networking API
 	quai.p2p = p2p
+	// Subscribe to the Blocks subscription
+	quai.p2p.Subscribe(config.NodeLocation, &types.WorkObject{})
+	quai.p2p.Subscribe(config.NodeLocation, common.Hash{})
+	quai.p2p.Subscribe(config.NodeLocation, &types.Transaction{})
```

Comment:
Does this happen before you set the backend? Im concerned that you've moved where we do our subs

Reply:
this should be removed, i agree, i didnt see